dev/io.openliberty.jakarta.validation.v31_fat/test-applications/val31/src/val31/web/Person.java

```diff
@@ -10,22 +10,17 @@
  *******************************************************************************/
 package val31.web;
 
-import jakarta.validation.Valid;
 import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
 
 /**
  *
  */
 public record Person(@NotNull String name) {
 
-    @Valid
-    public boolean checkLength() {
-        String val = this.name;
-        boolean check = false;
-        int x = val.length();
-        if (x > 2 && x <= 12) {
-            check = true;
-        }
-        return check;
+    @Size(min = 6)
+    public String getPersons() {
```

Comment:
`getName` would be better for this, since it doesn't return a `Person`

Reply:
updated